src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25634.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue25634 : _IssuesUITest
+{
+	public Issue25634(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Picker ItemDisplayBinding doesn't support MVVM properly";
+
+	[Test]
+	[Category(UITestCategories.Picker)]
+	public void VerifyPickerItemDisplayBindingValue()
+	{
+		App.WaitForElement("PickerButton");
+		App.Tap("PickerButton");
+		VerifyScreenshot();
```

Comment:
Pending snapshot in all the platforms. Already available in the latest build.
![image](https://github.com/user-attachments/assets/8f8b4098-df53-489d-8430-b4a3912c1cef)
Could you commit the images?